src/main/java/com/fasterxml/jackson/databind/cfg/DeserializerFactoryConfig.java

```diff
@@ -52,12 +54,20 @@
      */
     protected final ValueInstantiators[] _valueInstantiators;
 
+    /**
+     * The validator that knows which POJO types can be deserialized securely.
+     * Used to check whether a desirialized type is dangerous or undesired. The 
+     * default sub-type validator blocks deserialization widgets, i.e. classes that
+     * are known to be dangerous when deserialized from untrusted sources.
+     */
+    protected final SubTypeValidator _subTypeValidator;
```

Comment:
Should we support an array of `SubTypeValidators` instead?
That would be more in line with the other configuration options in this class.
Moreover, it could help ensuring that the default `SubTypeValidator` cannot be bypassed.

Reply:
No, just one is enough. But I do have other concerns.